test/create_account.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
It's best to do `set -ex` first (see more here https://explainshell.com/explain?cmd=set+-ex).  `-x` is kinda matter of situation, but without `-e` script won't fail CI properly.

Reply:
done